osg_configure/configure_modules/rsv.py

```diff
@@ -774,7 +771,7 @@ def _validate_host_list(self, hosts, setting):
                          level=logging.ERROR)
                 ret = False
 
-            if port and re.search('\D', port):
+            if port and re.search('[^0-9]', port):
```

Comment:
is `[^0-9]` just for clarity, over `\D` ?

Reply:
I think I may have misread that line as `\d`, thought to myself "wait that's gotta be broken" and changed it to something more explicit.

Reply:
My favorite is when people write `[\d\D]` to mean `.` with the `re.S` flag  ;)